androidApp/build.gradle.kts

```diff
@@ -26,6 +34,10 @@ android {
     buildTypes {
         getByName("release") {
             isMinifyEnabled = false
+            signingConfig = signingConfigs.getByName("confetti")
+        }
+        getByName("debug") {
+            signingConfig = signingConfigs.getByName("confetti")
```

Comment:
Changing the keystore here means you'll have to remove your current debug app before installing a new one